I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/4.rpy", line 214, in script
    show nico neutral behind ocular at Position(xpos=485, xanchor=0, ypos=84,
IOError: Couldn't find file 'sfx_footsteps_enter.ogg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/4.rpy", line 214, in script
    show nico neutral behind ocular at Position(xpos=485, xanchor=0, ypos=84,
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\ast.py", line 1375, in execute
    renpy.exports.with_statement(trans, paired)
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\exports.py", line 1578, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\display\core.py", line 2240, in do_with
    clear=clear)
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\display\core.py", line 2690, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\display\core.py", line 3110, in interact_core
    renpy.audio.audio.interact()
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\audio\audio.py", line 1066, in interact
    periodic()
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\audio\audio.py", line 1019, in periodic
    six.reraise(exc[0], exc[1], exc[2])
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\audio\audio.py", line 997, in periodic_thread_main
    periodic_pass()
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\audio\audio.py", line 938, in periodic_pass
    c.periodic()
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\audio\audio.py", line 420, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\audio\audio.py", line 72, in load
    rv = renpy.loader.load(fn)
  File "C:\Users\Cornelius\Desktop\renpy-7.2.2-sdk\renpy\loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'sfx_footsteps_enter.ogg'.

Windows-8-6.2.9200
Ren'Py 7.2.2.491
Get it Right the First Time 0.7
Sun Jun 02 03:11:23 2019
